library/tasks/ZarrUtils.wdl

```diff
@@ -93,12 +93,12 @@ task OptimusZarrConversion {
   command {
     set -euo pipefail
 
-
-    if ~{counting_mode} == "sc_rna"
-    then
+    if [ "${counting_mode}" == "sc_rna" ]; then
         EXPRESSION_DATA_TYPE_PARAM="exonic" 
+        ADD_EMPTYDROPS_DATA="yes"
```

Comment:
Can we use boolean values instead of strings here?

Reply:
We could but the python script, for which this variable is prepared, requires strings as argument options.